phi/eval/eval.py

```diff
@@ -5,18 +5,25 @@
 from pydantic import BaseModel, ConfigDict, field_validator, Field
 
 from phi.agent import Agent, RunResponse
+from phi.model.base import Model
 from phi.utils.log import logger, set_log_level_to_debug
 from phi.utils.timer import Timer
 
 
 class AccuracyResult(BaseModel):
     score: int = Field(..., description="Accuracy Score between 1 and 10 assigned to the Agent's answer.")
     reason: str = Field(..., description="Detailed reasoning for the accuracy score.")
+    relevance: int = Field(..., description="Relevance Score between 1 and 10 assigned to the Agent's answer.")
+    completeness: int = Field(..., description="Completeness Score between 1 and 10 assigned to the Agent's answer.")
```

Comment:
Maybe explain why these are different from `score`? Perhaps some examples will be useful.